src/pages/home/report/ReportActionsView.js

```diff
@@ -284,8 +280,8 @@ class ReportActionsView extends React.Component {
                 reportID={this.props.reportID}
                 action={item.action}
                 displayAsGroup={this.isConsecutiveActionMadeByPreviousActor(index)}
-                shouldDisplayNewIndicator={this.initialNewMarkerPosition > 0
-                    && item.action.sequenceNumber === this.initialNewMarkerPosition}
+                shouldDisplayNewIndicator={this.props.report.newMarkerSequenceNumber > 0
+                    && item.action.sequenceNumber - 1 === this.props.report.newMarkerSequenceNumber}
```

Comment:
Why `-1` ?

Reply:
Because `this.props.report.maxSequenceNumber - this.props.report.unreadActionCount` has an offset of 1, but you are right. I just changed `this.props.report.maxSequenceNumber - this.props.report.unreadActionCount` to `this.props.report.maxSequenceNumber - this.props.report.unreadActionCount + 1`, which makes more sense

Reply:
Ok I'm still a little confused then... why do we care what the `unreadActionCount` is? Shouldn't `this.props.report.newMarkerSequenceNumber` be the single source of truth for whether to show the indicator or not? If we don't want it to show we can just set it to `0`?

Reply:
Yep! Sorry if I didn't make myself clear:
- `this.props.report.newMarkerSequenceNumber` is now the single source of truth
- When we set it, we need to set it to `(this.props.report.maxSequenceNumber - this.props.report.unreadActionCount) + 1`